src/app/components/activity/feed/view.ts

```diff
@@ -277,7 +295,7 @@ export class ActivityFeedView {
 		Analytics.trackEvent('activity-feed', 'loaded-more', 'page-' + this.totalTimesLoaded);
 	}
 
-	async loadNew(newCount = ItemsPerPage) {
+	async loadNew(newCount = this.itemsPerPage) {
```

Comment:
matter of preference but i dont like using these. can we just default to null and use a null coalesce in the method body?

Reply:
Ehh, would you do something like
```
newCount = newCount ?? this.itemsPerPage;
```
?

Reply:
yeah